CMakeLists.txt

```diff
@@ -659,12 +659,12 @@ if(NOT gnutls)
 			TYPE RECOMMENDED
 			PURPOSE "Provides HTTPS support to libtorrent"
 	)
-	if(OPENSSL_FOUND)
+
+	if(TARGET OpenSSL::SSL)
```

Comment:
@FranciscoPombal would you mind making sure I merged this correctly from RC_1_2?
The new feature in master is support for using GnuTLS instead of OpenSSL